src/Columns/ColumnObject.cpp

```diff
@@ -30,6 +30,23 @@ const std::shared_ptr<SerializationDynamic> & getDynamicSerialization()
     return dynamic_serialization;
 }
 
+struct ColumnObjectCheckpoint : public ColumnCheckpoint
+{
+    using CheckpointsMap = std::unordered_map<String, ColumnCheckpointPtr>;
```

Comment:
I think we can use `std::string_view` here as key. All these keys are already stored in `typed_paths/dynamic_paths`, so we can avoid storing the same data twice